modules/playban/src/main/PlaybanFeedback.scala

```diff
@@ -1,25 +1,25 @@
 package lila.playban
 
+import lila.core.i18n.I18nKey.site as trans
+
 final private class PlaybanFeedback(
     chatApi: lila.core.chat.ChatApi,
     lightUser: lila.core.LightUser.Getter
 )(using Executor):
 
-  private val tempBan = "may lead to temporary playing restrictions."
-
-  def abort(pov: Pov): Unit = tell(pov, s"Warning, {user}. Aborting too many games $tempBan")
+  def abort(pov: Pov): Unit = tell(pov, s"i18n.${trans.playbanFeedbackAbort} {user}")
 
-  def noStart(pov: Pov): Unit = tell(pov, s"Warning, {user}. Failing to start games $tempBan")
+  def noStart(pov: Pov): Unit = tell(pov, s"i18n.${trans.playbanFeedbackNoStart} {user}")
```

Comment:
suddenly we're sending i18n keys to the chat, were before there only was human-readable text.

this means that all clients displaying chats must now support these i18n keys.

the 2 mobile apps currently don't, and will instead show `i18n.playbanFeedbackNoStart` to the end user.

Reply:
Noted.

As we can't translate on the fly, it seems that there are only 2 options left:

1. Keep Lichess messages in English in the chat
2. Send the translated text (2 messages if the 2 players speak different languages)